legacy/src/main/java/org/opensearch/sql/legacy/plugin/RestSqlAction.java

```diff
@@ -155,10 +155,9 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
                             if (newSqlRequest.isExplainRequest()) {
                                 LOG.info("Request is falling back to old SQL engine due to: " + exception.getMessage());
                             }
-                            LOG.debug("[{}] Request {} is not supported and falling back to old SQL engine",
```

Comment:
Why was this changed?

Reply:
Removed this because we are forcing it to throw an exception instead of falling back to legacy.